src/popup/pages/JwtSign.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <IonPage />
+</template>
+
+<script lang="ts">
+import { IonPage } from '@ionic/vue';
+import { defineComponent, onMounted } from 'vue';
+import { useRoute } from 'vue-router';
+import { Encoded, decode, signJwt } from '@aeternity/aepp-sdk';
+
+import { AeAccountHdWallet } from '@/protocols/aeternity/libs/AeAccountHdWallet';
+import { JWT_HEADER, MODAL_CONFIRM_UNSAFE_SIGN, PROTOCOLS } from '@/constants';
+import { handleUnknownError, toBase64Url } from '@/utils';
+import { RejectedByUserError } from '@/lib/errors';
+import {
+  useAccounts,
+  useAeSdk,
+  useDeepLinkApi,
+  useModals,
+  useUi,
+} from '@/composables';
+
+export default defineComponent({
+  name: 'JwtSign',
+  components: {
+    IonPage,
+  },
+  setup() {
+    const route = useRoute();
+
+    onMounted(async () => {
+      const { callbackOrigin, openCallbackOrGoHome } = useDeepLinkApi();
+      const { nodeNetworkId } = useAeSdk();
+      const { openModal } = useModals();
+      const { setLoaderVisible } = useUi();
+      const { getLastActiveProtocolAccount } = useAccounts();
+
+      try {
+        setLoaderVisible(true);
+        const payload = route.query.payload?.toString();
+        const jwk = route.query.jwk?.toString() === 'true';
```

Comment:
I wouldn't make this a user choice, lets always include sub_jwk as its a self signed jwt